### PR TITLE
Remove BV calculation options

### PR DESCRIPTION
**Pull request by SJuliez**
After @HammerGS 's poll and discussion, this PR removes all unofficial BV calc modifications. In that poll thread, I think monbvol stated it well: 

"As long as the end result is a single value when doing so there's going to be units that just aren't going to measure up one way or another and all adjusting how the single final value is going to do is change in what way and possibly which units do not measure up."

### CODE REVIEW

**Review by HammerGS — APPROVED** [review]
Fully support this.

File: megamek/src/megamek/common/battlevalue/JumpShipBVCalculator.java [hunk 1]
```diff
@@ -463,17 +463,8 @@ public static int calculateBV(Jumpship jumpShip, boolean ignoreC3, boolean ignor
...
-            if (finalBV == 0) {
-                finalBV = dbv + obv;
-            }
-            bvReport.addLine("2 * sqrt(Offensive BV * Defensive BV", "2 * sqrt(" + obv + " + " + dbv + ")", "");
-        } else {
-            finalBV = dbv + obv;
-            bvReport.addLine("Offensive BV + Defensive BV", obv + " + " + dbv, "");
-        }
+        double finalBV = dbv + obv;
+        bvReport.addLine("Offensive BV + Defensive BV",obv + " + " + dbv, "");
```
Comment (Windchild292) [inline megamek/src/megamek/common/battlevalue/JumpShipBVCalculator.java:467]:
```suggestion
        bvReport.addLine("Offensive BV + Defensive BV", obv + " + " + dbv, "");
```

**Review by Windchild292 — CHANGES_REQUESTED** [review]
This is missing unit testing removals.

**Comment by Windchild292** [comment]
There's still a failure in megamek.common.CrewTest > testGetBVSkillMultiplier at CrewTest.java:93.

**Review by Windchild292 — APPROVED** [review]

**Pull request merged**